h/static/styles/partials/_search.scss

```diff
@@ -95,6 +94,7 @@
 }
 
 .search-result__timeframe {
+  color: $grey-4;
```

Comment:
This change wasn't mentioned in the description. Is it intended?

Reply:
Good catch! I actually do need to make this change but kinda missed it when I did the diff so I update the commit message to reflect this change as well.